src/pages/signup/index.tsx

```diff
@@ -35,13 +38,13 @@ const Signup = () => {
   const [userType, setUserType] = useState('user');
   const [passwordVisible, setPasswordVisible] = useState(false);
   const [confirmPasswordVisible, setConfirmPasswordVisible] = useState(false);
+  const [isSubmitted, setIsSubmitted] = useState(false);
```

Comment:
not sure what's the point of isSubmitted?

Reply:
to ensure that the therapist won't submit the form after receiving the message to check his email since we don't navigate the therapist to any page

Reply:
Tmam, Got it